ErlangInstaller.xcodeproj/project.pbxproj

```diff
@@ -442,7 +446,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				MACOSX_DEPLOYMENT_TARGET = 10.10;
+				MACOSX_DEPLOYMENT_TARGET = 10.13;
```

Comment:
Notice this has incremented - will this prevent installing on earlier versions (10.10) of OSX. I don't really care, because I always stay up to date, and it's best to do so, but just to be certain, is the change necessary ?

Reply:
Apple made too many changes to Swift so compiling the current master commit on 10.11 was notably error prone (>200+).
As far as I have checked, the version 1.0.2 of the installer works fine for 10.10 to 10.12 - apart from the "menu-bar-icon" appearance on dark mode. So, perhaps the only thing that needs to be changed is that the icon file be moved to Assets.xcassets folder and changed to template type.
I deliberately changed the deployment to 10.13 to make sure everything was up to date, including the Swift compiler.
I can try and change the deployment target to 10.12. 
One other thought: On our downloads page we offer the latest version of the installer. Might I suggest we offer Mac users to download the version of the installer best suited to their version of macOS?

Reply:
Yep. I just changed the deployment target to 10.10 - running on a 10.13.5 OSX VM, it built and ran ok. Is this just advisory, or does the XCode toolchain do any API checking to ensure it will run sucessfully on 10.10. This is not a big deal for me. People tend to run on ancient versions (in which case half their applications will be broken) or the latest version (so they can have the latest security patches and product improvements). Quick check here at the office, all the developers are on 10.13.*, random business manager is on 10.9.*. I think installing the latest patch to OSX is acceptable (for this major Swift API upgrade) - it's not like we're going to be enforcing it on every release @HernanRivasAcosta. @nacicatus - I don't want to put too many obstacles in the way of getting your PR merged promptly. Both, thoughts?

Reply:
I would ideally like to get this PR merged before 5pm BST.

Reply:
Thanks @bryanhuntesl  for the vote of confidence and@HernanRivasAcosta for the critique :smile:

Reply:
No worries, interesting for me as I know so little about Swift. Thank you!